### DISCUSSION TITLE
How to customize zeek scripts in securityonion2.3

### DISCUSSION THREAD

**Discussion by SafelineMan** [Security-Onion-Solutions/securityonion#2011] [Unsupported Versions] [answered]
when I  customize zeek scripts，it cant work

[onion@securityonion logs]$ cat console.log 
fatal error in /opt/zeek/share/zeek/site/local.zeek, line 3: can't find bzar

**Comment by SafelineMan** [comment]
dididididi

**Comment by TOoSmOotH** [comment]
Not sure how we can help you if you don't put any details in your post.

**Comment by SafelineMan** [comment]
According to the document, I put the custom script into /opt/so/conf/zeek/
![image](https://user-images.githubusercontent.com/33274442/100210574-1c10a080-2f46-11eb-849d-1cb8bbecda08.png)

![image](https://user-images.githubusercontent.com/33274442/100210875-6eea5800-2f46-11eb-8f99-c2ca808a0890.png)

But he can't take effect
Do I need an editor local.zeek

![image](https://user-images.githubusercontent.com/33274442/100211057-a0632380-2f46-11eb-8090-75767c3bedaa.png)

**Comment by TOoSmOotH** [comment]
https://docs.securityonion.net/en/2.3/zeek.html?highlight=local.zeek#configuration

**Reply by SafelineMan** [reply]
thank you very much, But when I configure zeek like this，And put the bzar script in the corresponding directory，There's a problem，how i slove it
![image](https://user-images.githubusercontent.com/33274442/100301027-38f2b580-2fd2-11eb-892f-b315a1e429dd.png)

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/zeek.html#custom-scripts has been updated. Please try the new guidance there.

**Reply by SafelineMan** [reply]
thankyou very much  it works
